php/EE/Migration/Containers.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace EE\Migration;
+use EE\RevertableStepProcessor;
+use function EE\Utils\default_launch;
+use EE;
+
+/**
+ * Migrates existing containers to new image
+ */
+class Containers {
+
+	/** @var RevertableStepProcessor $rsp Keeps track of  */
+	private static $rsp;
+
+	public static function start_container_migration() {
```

Comment:
I think I've asked you to add a function doc for all functions. Please add comments for each function.

https://github.com/EasyEngine/easyengine/pull/1134/commits/dda2733b4a7b4a3e6fb13f070cefadd752f5df90#r206098226